packages/jaeger-ui/src/components/common/DetailsCard/DetailTableDropdown.tsx

```diff
@@ -108,7 +107,7 @@ export default class DetailTableDropdown extends React.PureComponent<TProps> {
               }
             >
               <Button className="DetailTableDropdown--Btn Apply" onClick={confirm}>
-                <FaCheck size={18} />
+                <IoCheckmarkCircle size={18} />
```

Comment:
I think this should be `IoCheckmark`, as it visually matches with `FaCheck`

Reply:
You are right, I fixed this.